src/xcode/ENA/ENA/Source/Workers/Keychain/KeychainHelper.swift

```diff
@@ -96,16 +96,16 @@ struct KeychainHelper {
 	/// Generates and stores a new random database key
 	/// - Throws: a `KeychainError` in case the generation or database save fails
 	/// - Returns: the newly created key
-	func generateContactDiaryDatabaseKey() throws -> String {
-		var bytes = [UInt8](repeating: 0, count: 32)
-		let result = SecRandomCopyBytes(kSecRandomDefault, bytes.count, &bytes)
-		guard result == errSecSuccess else {
-			Log.error("Error creating random bytes.", log: .api)
-			throw KeychainError.keyGenerationFail
-		}
-
-		let key = "x'\(Data(bytes).hexEncodedString())'"
-		try saveToKeychain(key: ContactDiaryStore.encriptionKeyKey, data: Data(key.utf8))
-		return key
-	}
+//	func generateContactDiaryDatabaseKey() throws -> String {
+//		var bytes = [UInt8](repeating: 0, count: 32)
+//		let result = SecRandomCopyBytes(kSecRandomDefault, bytes.count, &bytes)
+//		guard result == errSecSuccess else {
+//			Log.error("Error creating random bytes.", log: .api)
+//			throw KeychainError.keyGenerationFail
+//		}
+//
+//		let key = "x'\(Data(bytes).hexEncodedString())'"
+//		try saveToKeychain(key: ContactDiaryStore.encryptionKey, data: Data(key.utf8))
+//		return key
+//	}
```

Comment:
Ist das Kunst oder kann das weg? ;)
```suggestion
```

Reply:
Still needed once the store is added back to the target